nodepool/elements/cache-files/source-repository-images

```diff
@@ -0,0 +1,2 @@
+CentOS-8-x86_64-1905-dvd1.iso file /opt/cache/files/CentOS-8-x86_64-1905-dvd1.iso http://isoredirect.centos.org/centos/8/isos/x86_64/CentOS-8-x86_64-1905-dvd1.iso
```

Comment:
I guess you cannot write it directly to /srv/share/isos/centos.iso?

Reply:
we could but we should version things, for stable branch which could be different